.travis.yml

```
@@ -4,6 +4,7 @@ go:
   - "1.10"
   - "1.11"
   - "1.12"
+  - "1.13"
```

Let's drop 1.10 and 1.11 because they're no longer supported. Also, in Travis, 1.13 means exactly 1.13, we probably want "1.13.x":

```suggestion
  - "1.13.x"
```
  
Should probably do that for 1.12.x as well.

(Alternatevly, "1.x" can be used for "latest stable")